draft-ietf-httpbis-digest-headers.md

```diff
@@ -338,9 +338,9 @@ The Internet Assigned Numbers Authority (IANA) acts as a registry for
 digest-algorithm values.
 The registry contains the tokens listed below.
 
-Some algorithms, although registered, have since been found vulnerable:
+Some digest-algorithms, although registered, rely on vulnerable algorithms:
 the MD5 algorithm MUST NOT be used due to collision attacks [CMU-836068]
-and the SHA algorithm MUST NOT be used due
+and the SHA-1 algorithm MUST NOT be used due
```

Comment:
If we want to reference the crypto algorithm, SHA-1 is the right name.

Generally, with this statement we explicitly forbid using SHA-1 and MD5 based digest-algorithms, even if not listed here (that is: a future  multihash digest-algorithm with https://github.com/multiformats/multicodec/blob/master/table.csv#L71)